README.md

```diff
@@ -56,6 +56,51 @@ We provide the following ways for you to use xrefcheck:
 
 If none of those are suitable for you, please open an issue!
 
+If you want some external links to not be verified, you can use one of the following ways to ignore that links:
+1. Add the regular expression that match ignoring link to optional `ignoreRefs` parameter of your config file.
+
+For example: 
+```sh
+ignoreRefs:
+  - https://bad.reference.(ru|com)(/?)
+```
+allows to ignore both `https://bad.reference.ru` and `https://bad.reference.com` with or without last "/".
+
+2. Add right in-place annotation using one of the following ignoring mode (each mode is just a comment with a certain syntax).
+
+* Ignore the link:
+
+```sh
+<!-- xrefcheck: ignore link -->
+[Bad 'ru' reference](https://bad.reference.ru)
+[Bad 'com' reference](https://bad.reference.com)
+```
+
+Therefore only `https://bad.reference.ru` will be ignored.
+
+* Ignore the paragraph:
+
+```sh
+<!-- xrefcheck: ignore paragraph -->
+[Bad 'ru' reference](https://bad.reference.ru)
+[Bad 'com' reference](https://bad.reference.com)
+
+[Bad 'io' reference](https://bad.reference.io)
+```
+
+In this way `https://bad.reference.ru` and `https://bad.reference.com` will be ignored and `https://bad.reference.io` will still be verified.
+
+* Ignore the whole file:
+```sh
+<!-- a comment -->
+<!-- another comment -->
+
+<!-- xrefcheck: ignore file -->
+...the rest of the file...
+```
+
+Using this you are able to ignore the whole file besides  comments before `<!-- xrefcheck: ignore file -->`.
+
 To find all broken links in a repository, run from within its folder:
```

Comment:
Is overall this looks good, nice that we have this documented. 

Usually, we apply some spell-checker to public documents, please use some checker to proof-read the document (e.g. grammarly.com)